dubbo-serialization/dubbo-serialization-msgpack/src/test/java/org/apache/dubbo/common/serialize/msgpack/BizException.java

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Please add Apache License header for this file

Reply:
for test @xiaoheng1